model/task/task_log.go

```diff
@@ -180,14 +180,12 @@ func getLogService(ctx context.Context, o TaskLogOutput) (log.LogService, error)
 	return log.NewLogServiceV0(b), nil
 }
 
-// getBucketConfigForProject returns the appropriate bucket config for a project,
-// using long retention bucket if the project is in the long retention list.
-func getBucketConfigForProject(project string, originalBucketConfig evergreen.BucketConfig) evergreen.BucketConfig {
+// getBucketConfigForProject returns the appropriate bucket config for a project, using long
+// retention bucket if the project is in the long retention list. It returns a boolean indicating if the original bucket is being used.
+func getBucketConfigForProject(project string, originalBucketConfig evergreen.BucketConfig) (evergreen.BucketConfig, bool) {
```

Comment:
I might of skipped it, but do we still need the bool? I looked at the callers but they all ignore it

Reply:
good catch, I must have refactored but left it behind 🙏🏻